modules/media/Setup.md

```diff
@@ -0,0 +1,27 @@
+## Media module set up 
+
+Set the `mdiaPath` config value to the desired path on the server where the files 
+will be uploaded.
+
+Create the directory on the server.
+
+The upload path must be readable and writable by your web server; 
+either the web server `user` or `group` must have read and write permissions.
+
+The default group for your web server process is listed below
+```
+Ubuntu: $GROUP$ = www-data
+CentOS: $GROUP$ = apache
+```
+
+To find the `user` of your web server, run 
+`ps aux | grep 'apache' | egrep -v 'grep|Ss' | awk '{ print $1 }' | sort | uniq`
+To find the `group` of your web server, run 
+`ps aux | grep 'apache' | egrep -v 'grep|Ss' | awk '{ print $1 }' | sort | uniq | groups`
+
```

Comment:
Doesn't seem to work correctly in my VM, I get my own group.

last item should be `xargs groups`

Reply:
the commands seem to be working for me
which OS are you on ?

Reply:
ubuntu 16.04

Reply:
no indeed you are correct